examples/bzlmod/entry_point/BUILD.bazel

```diff
@@ -1,4 +1,4 @@
-load("@pip//:requirements.bzl", "entry_point")
+load("@pip_39//:requirements.bzl", "entry_point")
```

Comment:
I don't think this is correct

This loads the version-specific macro, which means anything it resolves will resolve to a 3.9 dependency. In comparison, the rules will using the default Python, which may not be 3.9.

But, I'm guessing this was done because `entry_point` isn't implemented in the multi-pip variant of the rules?

Reply:
Correct this is because we don’t have the pip hub working yet. As well as you can still access the python specific version